docs/pages/graph.rst

```diff
@@ -0,0 +1,77 @@
+.. _graph:
+
+GraphQL API
+===========
+
+Classes and functions to work with metadata for resources stored in the
+Rune platform. Metadata is fetched from Rune's GraphQL API (https://graph.runelabs.io/graphql),
+using a :class:`~runeq.v2sdk.client.GraphClient`.
```

Comment:
Can you do a find & replace in this file, to update `runeq.v2sdk` to `runeq.resources`?

```suggestion
using a :class:`~runeq.resources.client.GraphClient`.
```